aten/src/ATen/native/native_functions.yaml

```diff
@@ -1796,7 +1796,7 @@
   dispatch:
     CPU, Meta: resize_
     CUDA: resize_cuda_
-    QuantizedCPU: quantized_resize_cpu_
+    QuantizedCPU, QuantizedCUDA: quantized_resize_cpu_
```

Comment:
following naming convention for quantized functions, this should be resize_quantized_ I think